config/stage.json

```diff
@@ -94,7 +94,8 @@
 		"upgrades/domain-search": true,
 		"upgrades/in-app-purchase": false,
 		"upgrades/paypal": true,
-		"upgrades/premium-themes": true
+		"upgrades/premium-themes": true,
+		"upgrades/removal-survey": true
```

Comment:
I would suggest holding off on enabling stage until we're also ready to turn a feature on in prod.